aggregator/src/aggregator/collection_job_driver.rs

```diff
@@ -1154,7 +1189,7 @@ mod tests {
         // Batch aggregations indicate not all aggregation jobs are complete, and there is an
         // unaggregated report in the interval.
         collection_job_driver
-            .step_collection_job(Arc::clone(&ds), Arc::clone(&lease))
+            .step_collection_job(Arc::clone(&ds), clock.clone(), Arc::clone(&lease))
```

Comment:
Elsewhere, I think we make a point of using `Arc<C>` and cloning that, just in case a particular `Clock` impl happens to be big... but honestly, it's been years now, and the only `Clocks` we have are `MockClock` (which is just an `Arc<Mutex<Time>>` and so roughly pointer sized and already provides interior mutability) or `RealClock` which I think is an empty struct that just calls `Instant::now`. So  think we'd save ourselves some hassle throughout Janus by having all `Clock`s be `Copy`.

Anyway `clock.clone()` is fine for this PR, but I wanted to offer some context.

https://github.com/divviup/janus/issues/3948

Reply:
Yeah, what's here is what we should continue to do. I was just wrong.